### DISCUSSION TITLE
In PPP kinematic orbit determination, where are the differences in the source code between the orbit determination calculations using a single GPS system and a single Galileo system?

### DISCUSSION THREAD

**Discussion by MinSudo** [groops-devs/groops#272] [Q&A]
In GROOPS, when using the PPP kinematic module for LEO satellite positioning, why is there a significant error (meter-level) in the orbit determination results when using data from a single GPS system, whereas the error is much smaller (centimeter-level) when using data from a single Galileo system? The impact of errors caused by precise products (sp3, clk, bias) or other system errors (such as antenna hardware calibration, center of mass calibration) has already been ruled out. All other parameters in the XML file are set the same. I would like to know if there are differences between GPS and Galileo, and whether the processing steps in the GnssProcessingStep::State::estimateSolution function in the source code are handled differently.

**Comment by tmayerguerr** [comment]
Hi MinSudo,
GPS and Galileo are treated in exactly the same way in GROOPS. It is probably still due to the metadata (bias etc).
regards
Torsten
